config.go

```diff
@@ -438,6 +445,18 @@ func (s *Settings) GetGithubOauthToken() (string, error) {
 	return "", errors.New("no github token in settings")
 }
 
+func (s *Settings) SetUpNewRelic() (*newrelic.Application, error) {
+	if s.NewRelic.ApplicationName == "" || s.NewRelic.LicenseKey == "" {
+		return nil, nil
+	}
+	config := newrelic.NewConfig(s.NewRelic.ApplicationName, s.NewRelic.LicenseKey)
+	app, err := newrelic.NewApplication(config)
+	if err != nil || app == nil {
+		return nil, errors.Wrap(err, "error creating New Relic application")
+	}
+	return &app, err
```

Comment:
why are we making it a pointer? how does the second conditional of 454 work then?)

Reply:
Fixed. The docs promise that if app is nil, err will not be nil, so the nil check on 454 is a sanity check.